krummstab/__init__.py

```diff
@@ -687,7 +699,8 @@ def validate_marks_json(_the_config: config.Config) -> None:
             f"There are missing points in the '{marks_json_file.name}' file!"
         )
     if not all(
-        (float(mark) / _the_config.min_point_unit).is_integer() for mark in marks_list
+        (float(mark) / _the_config.min_point_unit).is_integer()
+        for mark in marks_list if mark != "Plagiat"
```

Comment:
It would be nice to define the string `Plagiat` only once globally, together with the other string constants. It also appears a couple of times in `summaries.py`. I think we should also mention in the README that this magic string exists and how it can/should be used.

Reply:
Maybe also switch it to English because we use English everywhere else and may have English-speaking tutors.